src/main/java/org/broadinstitute/consent/http/service/DacService.java

```diff
@@ -306,13 +306,13 @@ private boolean isAuthUserChairOrMember(AuthUser authUser) {
     /**
      * Filter data access requests by the DAC they are associated with.
      */
-    List<DataAccessRequest> filterDataAccessRequestsByDac(List<DataAccessRequest> documents, AuthUser authUser) {
-        if (isAuthUserAdmin(authUser)) {
+    List<DataAccessRequest> filterDataAccessRequestsByDac(List<DataAccessRequest> documents, User user) {
+        if (DarUtil.hasUserRole(user, 4)) {
             return documents;
         }
         // Chair and Member users can see data access requests that they have DAC access to
-        if (isAuthUserChairOrMember(authUser)) {
-            List<Integer> accessibleDatasetIds = dataSetDAO.findDataSetsByAuthUserEmail(authUser.getName()).
+        if (DarUtil.hasUserRole(user, 1) || DarUtil.hasUserRole(user, 2)) {
+            List<Integer> accessibleDatasetIds = dataSetDAO.findDataSetsByAuthUserEmail(user.getEmail()).
```

Comment:
We should either mimic what we're doing in `isAuthUserChairOrMember` with a new method like `isUserChairOrMember`, or reference the user roles enums instead of using the integer values of the role ids.